egs/cl_english/v1/run.sh

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env bash
+
+# Copyright 2021  Behavox (author: Hossein Hadian)
+# Apache 2.0
+
+# This script demonstrates one example of continual learning (CL)
+# using Learning Without Forgetting (LWF) and DenLWF methods.
+# This specific example is for transferring from a subset of Fisher English
+# data to CommonVoice Indian subset. Therefore it's an example of accent adaptation
+# using CL from American to Indian English.
+
+# See this paper for more info: https://arxiv.org/abs/2110.07055
+
+# To run this script, first prepare the Fisher train/test data under
+# data/fsh_train and data/fsh_dev and LM under data/local/lm.gz.
+# Also prepare the lang and dict under data/lang_nosp and
+# data/local/dict_nosp respectively. Additionally, prepare the indian subset of
+# Common Voice under data/cvi_train and data/cvi_test.
+# Finally run these commands one by one:
+
+
+# Note: You can change 10000 to 50000 to replicate the results in the paper.
+utils/subset_data_dir.sh data/fsh_train 10000 data/fsh_train_10hr
+utils/subset_data_dir.sh data/cvi_train 10000 data/cvi_train_10hr
+
+utils/combine_data_dir.sh data/comb_train data/cvi_train_10hr data/fsh_train_10hr
+
+# These train an standalone model for each dataset and the combined data:
+# Note that we use mono tree for the source model so that LWF can be applied efficiently
+local/run_pipeline.sh --softmax true --tree-affix mono --tree-opts \
+                      "--context-width=1 --central-position=0"  cvi_train_10hr
+local/run_pipeline.sh --softmax true --tree-affix mono --tree-opts \
+                      "--context-width=1 --central-position=0"  fsh_train_10hr
+local/run_pipeline.sh data/comb_train
+
+
+# Do simple fine-tuning (i.e. transfer learning)
+local/chain/run_finetune_1b.sh \
+  --src-tree-dir exp_fsh_train_10hr/chain/tree_sp_mono \
+  --dir exp_cvi_train_10hr/chain/tdnn_ft1b_fsh2cvi_sp \
+  --exp exp_cvi_train_10hr --train-set cvi_train_10hr \
+  --src-mdl exp_fsh_train_10hr/chain/tdnn1a_noiv_small_sp/final.mdl
+
+# Do LWF continual learing
+local/chain/run_lwf_clean_1a.sh \
+  --src-tree-dir exp_fsh_train_10hr/chain/tree_sp_mono \
+  --dir exp_cvi_train_10hr/chain/tdnn_lwf1a_fsh2cvi_sp \
+  --exp exp_cvi_train_10hr --train-set cvi_train_10hr \
+  --src-mdl exp_fsh_train_10hr/chain/tdnn1a_noiv_small_sp/final.mdl \
+  --lwf-scale 0.8
```

Comment:
Do you really recommend LWF, from the paper I think you recommend LWF?
My feeling is it might be better to comment out the regular LWF stuff and only keep the recommended part of
the recipe as the main path.

Reply:
Thanks for the review!
I'll fix the logs.
In all the experiments that we tried, DenLWF worked better than regular LWF but I kept it for reference. I can make it default here.